Frends.PDF.Create/Frends.PDF.Create.Tests/UnitTests.cs

```diff
@@ -0,0 +1,139 @@
+
+        private PageContentElement _header;
+        private PageContentElement _footer;
+        private PageContentElement _title;
+        private PageContentElement _paragraphContent;
+        private PageContentElement _tableContent;
+
+        private Options _options;
+
+        [SetUp]
+        public void TestSetup()
+        {
+
+            if (!Directory.Exists(_folder))
+            {
+                Directory.CreateDirectory(_folder);
+            }
+            var logoPath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"../../../Files/logo.png");
+            _header = new PageContentElement { ContentType = ElementType.Header, FontFamily = "Times New Roman", FontSize = 8, FontStyle = FontStyleEnum.Regular, LineSpacingInPt = 11, ParagraphAlignment = ParagraphAlignmentEnum.Right, SpacingAfterInPt = 0, SpacingBeforeInPt = 8, ImagePath = logoPath, HeaderFooterStyle = HeaderFooterStyleEnum.LogoText, BorderWidthInPt = 0.5, ImageHeightInCm = 0.5 };
+            _footer = new PageContentElement { ContentType = ElementType.Footer, FontFamily = "Times New Roman", FontSize = 8, FontStyle = FontStyleEnum.Regular, LineSpacingInPt = 11, ParagraphAlignment = ParagraphAlignmentEnum.Center, SpacingAfterInPt = 0, SpacingBeforeInPt = 8, HeaderFooterStyle = HeaderFooterStyleEnum.TextPagenum, BorderWidthInPt = 0.0 };
+            _title = new PageContentElement { ContentType = ElementType.Paragraph, FontFamily = "Times New Roman", FontSize = 16, FontStyle = FontStyleEnum.Bold, LineSpacingInPt = 11, ParagraphAlignment = ParagraphAlignmentEnum.Left, SpacingAfterInPt = 0, SpacingBeforeInPt = 8 };
+            _paragraphContent = new PageContentElement { ContentType = ElementType.Paragraph, FontFamily = "Times New Roman", FontSize = 11, FontStyle = FontStyleEnum.Regular, LineSpacingInPt = 11, ParagraphAlignment = ParagraphAlignmentEnum.Left, SpacingAfterInPt = 0, SpacingBeforeInPt = 8 };
+
+            var tablePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"../../../Files/ContentDefinition.json");
+            var tableDefinition = File.ReadAllText(tablePath);
+            _tableContent = new PageContentElement { ContentType = ElementType.Table, Table = tableDefinition };
+
+            _options = new Options { ThrowErrorOnFailure = true };
+        }
+
+        [TearDown]
+        public void TestTearDown()
+        {
+            Directory.Delete(_folder, true);
+        }
+
+        private Result CallCreatePdf(PageContentElement[] contents)
+        {
+            return PDF.Create(_fileProperties, _docSettings, new DocumentContent { Contents = contents }, _options);
+        }
+
+        [Test]
+        public void WritePDF()
+        {
+            _fileProperties.FileExistsAction = FileExistsActionEnum.Overwrite;
+
+            _header.Text = @"This is a header";

```

Comment:
Setting those contents for header-footer-title-text are redundant in this test, could we remove those, since they are irrelevant and could potentially confuse the reader?